pilot/lib/pilot/build_manager.rb

```diff
@@ -38,7 +36,11 @@ def list(options)
         config[:app_identifier] = ask("App Identifier: ")
       end
 
-      builds = app.all_processing_builds + app.builds
+      if config[:app_platform].to_s.length == 0
+        config[:app_platform] = ask("App Platform (ios, appletvos, osx): ")
```

Comment:
Yes we should probably do that.   Also side note when working on this I noticed that in some places the code uses tvOS, tvos, and appletvos,  I know that at points the api needs serious versions of those values.  Maybe we should look into a standardizer when asking for inputs? I see the same possibly happening with mac, macOS, and osx.